build.xml

```diff
@@ -73,7 +73,7 @@
           dir="extension"
           resultproperty="test.result"
           failonerror="true">
-      <arg line="--verbose ${cfx.binary} test"/>
+      <arg line="--verbose ${cfx.binary} --prefs test/testhelper.json test"/>
```

Comment:
Please add a comment that this can be removed once ESR38 has been desupported. Maybe add a reference to this PR.